Buildings/Experimental/NatVentControl/VAV/TStatReset.mo

```diff
@@ -0,0 +1,89 @@
+within Buildings.Experimental.NatVentControl.VAV;
+block TStatReset
+  "Resets VAV thermostat if natural ventilation mode is on"
+  Controls.OBC.CDL.Interfaces.BooleanInput natVenSig
+    "True if natural ventilation is active; false if not" annotation (Placement(
+        transformation(extent={{-140,10},{-100,50}}),  iconTransformation(
+          extent={{-140,-20},{-100,20}})));
+  Controls.OBC.CDL.Interfaces.RealOutput htgStpVAV "VAV heating setpoint"
+    annotation (Placement(transformation(extent={{100,-50},{140,-10}}),
+        iconTransformation(extent={{100,-70},{140,-30}})));
+  Controls.OBC.CDL.Interfaces.RealOutput clgStpVAV "Cooling setpoint for VAV"
+    annotation (Placement(transformation(extent={{100,-90},{140,-50}}),
+        iconTransformation(extent={{100,-30},{140,10}})));
+  Controls.OBC.CDL.Interfaces.RealInput htgStp "Heating setpoint from VAV"
```

Comment:
change the name to`THeaSet`. Also, how are the input setpoints (heating and cooling) specified? It would be better to update the comments to like `Heating setpoint specified by/according to ......`.